src/Refactoring-Core/RBPullUpMethodRefactoring.class.st

```diff
@@ -142,66 +149,68 @@ RBPullUpMethodRefactoring >> checkSuperSendsFromPushedUpMethods [
 { #category : #preconditions }
 RBPullUpMethodRefactoring >> checkSuperSendsFromSiblings [
 	| siblings |
-	siblings := class superclass subclasses reject: [:each | each = class].
+	siblings := superClass subclasses reject: [:each | each = class].
 	siblings do: [:aRBClass | self checkSiblingSuperSendsFrom: aRBClass]
 ]
 
 { #category : #preconditions }
 RBPullUpMethodRefactoring >> checkSuperclass [
 	| overrideSelectors |
 	overrideSelectors := selectors
-		select: [ :each | class superclass directlyDefinesMethod: each ].
+		select: [ :each | superClass directlyDefinesMethod: each ].
 	overrideSelectors := overrideSelectors
 		reject: [ :each | 
 			| myTree superTree |
 			myTree := class parseTreeFor: each.
-			superTree := class superclass parseTreeFor: each.
+			superTree := superClass parseTreeFor: each.
 			superTree equalTo: myTree exceptForVariables: #() ].
 	overrideSelectors isEmpty
 		ifTrue: [ ^ self ].
-	class superclass isAbstract
+	superClass isAbstract
 		ifFalse: [ self
 				refactoringError:
 					('Non-abstract class <2p> already defines <1p>'
 						expandMacrosWith: overrideSelectors asArray first
-						with: class superclass) ].
+						with: superClass) ].
 	overrideSelectors do: [ :each | self checkBackReferencesTo: each ]
 ]
 
-{ #category : #private }
-RBPullUpMethodRefactoring >> copyDownMethod: aSelector [
+{ #category : #initialization }
+RBPullUpMethodRefactoring >> class1 [
```

Comment:
I' not sure you use this method. If you don't use, please remove it. If you use it, please rename it to something better.

Reply:
Sure, I'll do